src/date-util/date-util.enum.ts

```diff
@@ -0,0 +1,12 @@
+export enum ISO8601Format {
+  'YYYY' = 'YYYY',
+  'YYYYMMDD' = 'YYYYMMDD',
+  'YYYY-MM' = 'YYYY-MM',
+  'YYYY-MM-DD' = 'YYYY-MM-DD',
+  'YYYY-MM-DDTHH:mm:ss' = 'YYYY-MM-DDTHH:mm:ss',
+  'YYYY-MM-DDTHH:mm:ss.SSS' = 'YYYY-MM-DDTHH:mm:ss.SSS',
+  'HH:mm:ss.SSS' = 'HH:mm:ss.SSS',
+  'HH:mm:ss' = 'HH:mm:ss',
+  'HH:mm' = 'HH:mm',
+  'HH' = 'HH',
+}
```

Comment:
이런 경우는 enum보다 type union을 쓰라는 Effective TS의 가이드가 있습니다.
```suggestion
export type ISO8601Format = 'YYYY' | 'YYYYMMDD' | 'YYYY-MM' | 'YYYY-MM-DD' | 'YYYY-MM-DDTHH:mm:ss' | 'YYYY-MM-DDTHH:mm:ss.SSS' | 'HH:mm:ss.SSS' | 'HH:mm:ss' | 'HH:mm' | 'HH';
```

그리고 enum이 아니라 type이 되었으므로 dete-util.type.ts 에 추가하면 될 것 같습니다

Reply:
아..! 그 전에 있던 format 논의의 결론이 이런 형태의 enum으로 가는줄 알았습니다. type으로 수정하겠습니다

Reply:
사실 그 때 아무 결론이 안났......